src/python/turicreate/test/test_image_similarity.py

```diff
@@ -12,8 +12,12 @@
 from turicreate.toolkits._internal_utils import _mac_ver
 import tempfile
 from . import util as test_util
-import numpy as np
+
 from turicreate.toolkits._main import ToolkitError as _ToolkitError
+from turicreate.toolkits.image_analysis.image_analysis import MODEL_TO_FEATURE_SIZE_MAPPING, get_deep_features
+
+import coremltools
```

Comment:
This is no longer getting used.

Reply:
Unfortunately due to the (brittle) way that we test the minimal version of TuriCreate, we are going to need to remove this unnecessary line. `coremltools` is not a dependency for the minimal version of our TuriCreate. So having it imported at a top level break our unit tests even though the tests in this file are not ran for the minimal version.

I believe removing this line should be the final required change for this pull request. All of our other internal tests are passing.

Reply:
Makes sense, will remove the `import coremltools` line and push changes